src/paperless_tesseract/parsers.py

```diff
@@ -45,13 +45,22 @@ def get_thumbnail(self):
         """
 
         out_path = os.path.join(self.tempdir, "convert.png")
+        gs_out_path = os.path.join(self.tempdir, "gs_out.png")
 
         # Run convert to get a decent thumbnail
+
+        # https://github.com/danielquinn/paperless/issues/447
+        # call gs first
+        cmd = ["gs", "-q", "-sDEVICE=pngalpha",
```

Comment:
Rather than calling `gs` directly, you should add a new setting `GHOSTSCRIPT_BINARY` or `GS_BINARY`, as done for [`CONVERT_BINARY`](https://github.com/the-paperless-project/paperless/blob/6d8e4ad6bf92094149e2335a8f30cb976475fe70/src/paperless/settings.py#L261).

There are at least the following places where it should be added (there might be more I have missed):

* [`settings.py`](https://github.com/the-paperless-project/paperless/blob/6d8e4ad6bf92094149e2335a8f30cb976475fe70/src/paperless/settings.py#L261)
* [`paperless.conf.example`](https://github.com/the-paperless-project/paperless/blob/6d8e4ad6bf92094149e2335a8f30cb976475fe70/paperless.conf.example#L248)
* [`parsers.py`](https://github.com/the-paperless-project/paperless/blob/master/src/paperless_tesseract/parsers.py#L31) (only retrieval of the setting itself)

Reply:
Resolved by https://github.com/the-paperless-project/paperless/pull/494/commits/dd0bbc48286addb0e68012a0e3ada8d3c576e24c